pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -123,6 +123,7 @@ func BuildKubeAPIserverOptions(masterConfig configapi.MasterConfig) (*kapiserver
 	server.SecureServing.ServerCert.CertKey.KeyFile = masterConfig.ServingInfo.ServerCert.KeyFile
 	server.InsecureServing.BindPort = 0
 
+	server.Authentication.Anonymous.Allow = false
```

Comment:
@deads2k ptal: I added this line to get rid of the `W0413 17:32:29.303436 41108 authentication.go:362] AnonymousAuth is not allowed with the AllowAll authorizer. Resetting AnonymousAuth to false. You should use a different authorizer` warning.